### PR TITLE
Feature/api calls

### PR DESCRIPTION
**Pull request by cbmackintosh**
### What Changed?
- Reworks the existing component architecture to work with network requests
- Adds new components to the Database menu
- Adds several network requests to the api calls file
- Successfully fetches ongoing and resolved incidents from the server and displays on the UI
- Renames and reorganizes component architecture, deletes some default files.

### What's Next?
- Test functionality with additional endpoints as they become available on the backend
- Make an Incident component that will display the chart and other incident details
- Make items in the ongoing/resolved incidents lists clickable to open up a `<Route path='/incident/:id component={Incident}>`
- Using the id in the url as props, configure additional network requests to fetch all incident details and personnel
- Use that data to compile the chart

### Known Issues?
- Most of the endpoints are not functioning yet and therefore haven't been tested.

### Test Coverage?
- All tests are passing but existing tests should be rewritten and new tests should be added to reflected updated functionality.

### CODE REVIEW

File: cypress/integration/quick-command-tests.js [hunk 1]
```diff
@@ -57,16 +59,19 @@ describe('QuickCommand', () => {
         .get('input[data-cy=incident-date]').type('2021-05-21')
         .get('textarea[data-cy=incident-summary]').type('5.4 Richter Scale quake has disrupted the village of Palo Alto')
         .get('button[data-cy=declare-submission]').click()
-      cy.get('h3[data-cy=incident-name]').contains('San Andreas Faultline Disruption')
-        .get('p[data-cy=incident-type]').contains('Earthquake')
-        .contains('2021-05-21')
+        // needs to be reworked since incidents are now declared via network request but the endpoint isn't working yet
+      // cy.get('h3[data-cy=incident-name]').contains('San Andreas Faultline Disruption')
```
Comment (percworld) [inline cypress/integration/quick-command-tests.js:63]: I can also work on getting tests going again.

File: src/Store/store.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import OngoingIncidentsReducer from '../components/OngoingIncidents/OngoingIncidentsSlice';
+import ResolvedIncidentsReducer from '../components/ResolvedIncidents/ResolvedIncidentsSlice';
+import SearchResultsReducer from '../components/DatabaseSearch/DatabaseSearchSlice';
+
+export const store = configureStore({
+  reducer: {
+    ongoingIncidents: OngoingIncidentsReducer,
+    resolvedIncidents: ResolvedIncidentsReducer,
+    searchResults: SearchResultsReducer
```
Comment (percworld) [inline src/Store/store.js:10]: Looks like we may have two store files now.  Let's figure out which is imported.

File: src/components/DatabaseSearch/DatabaseSearch.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  displayContactDets
+} from './DatabaseSearchSlice'
+
+const DatabaseSearch = () => {
+
+  const dispatch = useDispatch()
+
+  const searchResults = useSelector(state => state.searchResults.searchResults).map(result => {
+    return (
+      <div onClick={() => dispatch(displayContactDets(result.id))}>
```
Comment (percworld) [inline src/components/DatabaseSearch/DatabaseSearch.js:14]: Thank you this looks great- might rename Dets to details?

**Review by percworld — COMMENTED** [review]
This is i big PR! I am excited to see the development and where this is going.  Excellent use of useSealector and dispatch and extraReducers.

**Review by JeffShepherd — APPROVED** [review]
Solid work! This is a huge leap forward. Excited to link this to the chart view!

**Pull request merged**